rfd/0013-automating-bench-soak-tests.md

```diff
@@ -0,0 +1,61 @@
+# RFD 13 - Automating `tsh bench` soak tests and data collection
+## Purpose
+Automate the running of tests and collection of data (profiles and dashboards) for the soak test portion of the test plan.
+
+### Overview
+At the moment, `tsh bench` tool is used to manually perform and collect data for soak tests following commands against a node.
+
+Soak Tests    
+Run 4hour soak test with a mix of interactive/non-interactive sessions:   
+`tsh bench --duration=4h user@teleport-monster-6757d7b487-x226b ls`   
+`tsh bench -i --duration=4h user@teleport-monster-6757d7b487-x226b ps uax`
+
+## Technical Details
+### Cluster Setup
+Cluster setup and load tests will continue to be a manual process at the moment and is out of scope for this RFD.
+
+### Image generation
+A flag will be added to `tsh bench` that can be used to query and capture the test output to analyze. The test output will be exported as a snapshot of the dashboard after the given duration plus 20 minutes. 
+
+```
+// duration of test will be 4 hours and 20 minutes
+tsh bench \
+   --duration=4h \
+   --output-image=file.png \
```

Comment:
@klizhentas @benarent What do you think about adding this flag?

Reply:
We can also have a flag, `--snapshot`, and generate an image name with a timestamp. What do you all think about it?